okhttp/src/main/java/com/squareup/okhttp/internal/http/HttpEngine.java

```diff
@@ -413,18 +398,14 @@ public HttpEngine recover(IOException e, Sink requestBodyOut) {
 
     Connection connection = close();
 
+    // TODO(nfuller): I'm not convinced about the retention of RouteSelector in HttpEngine. Its use
```

Comment:
This doesn't really seem like a TODO; perhaps it's more of a bug report. What's the bug?